view.php

```diff
@@ -83,15 +83,17 @@
 }
 
 // Compose Moodle user object for host.
-$hostmoodleuser = new stdClass();
-$hostmoodleuser->firstname = $hostuser->first_name;
-$hostmoodleuser->lastname = $hostuser->last_name;
-$hostmoodleuser->alternatename = '';
-$hostmoodleuser->firstnamephonetic = '';
-$hostmoodleuser->lastnamephonetic = '';
-$hostmoodleuser->middlename = '';
-
-$meetinginvite = $service->get_meeting_invitation($zoom);
+if (!$showrecreate) {
```

Comment:
It seems like this the first chunk of code depends on $hostuser instead of $showrecreate:
```suggestion
if ($hostuser) {
```

Reply:
I think this change make any functional difference, but it is semantically more understandable.

I have pushed an improvement on https://github.com/ucla/moodle-mod_zoom/pull/245/commits/4d8f3a6dbcff4053978818325e85cd440a894546